docs/reference/manifest/appdomain.md

```diff
@@ -18,13 +21,17 @@ Specifies additional domains that load pages in the add-in window. It also lists
 ```
 
 > [!IMPORTANT]
-> 1. The value of the **AppDomain** element must include the protocol (e.g., `<AppDomain>https://myappdomain</AppDomain>`).
-> 2. Do *not* put a closing slash, "/", on the value.
+> 1. The value of the **AppDomain** element must include the protocol (e.g., `<AppDomain>https://myappdomain.com</AppDomain>`).
+> 2. If there is an explicit port for the domain, this must be included too (e.g.,`<AppDomain>https://myappdomain.com:9999</AppDomain>`).
+> 3. If a subdomain needs to be trusted, then it must be included too (e.g.,`<AppDomain>https://mysubdomain.myappdomain.com</AppDomain>`). The subdomain `mysubdomain.mydomain.com` and `mydomain.com` are different domains. If both need to be trusted, then both need to be in separate **AppDomain** elements.
+> 4. Listing the same domain as the one specified in the [SourceLocation element](sourcelocation.md) has no effect and may be misleading. In particular, when you are developing on `localhost`, you do not need to create an **AppDomain** element for `localhost`.
+> 5. Do not include any segments of a URL past the domain. For example, do not include the full URL of a page.
+> 6. Do *not* put a closing slash, "/", on the value.
```

Comment:
Decided to leave this because of the bold on the not. Do we need the bolding here?

Reply:
Yes. This caused problems in the past, so the PM wanted to emphasize it.